setup.py

```diff
@@ -471,8 +468,6 @@ def pxd(name):
 
 tseries_depends = np_datetime_headers + ['pandas/_libs/tslibs/np_datetime.pxd']
 
-# some linux distros require it
```

Comment:
hmm do we need this?

Reply:
Apparently not.  It has apparently sat there unused for a while.